Move jobsets multikueue run details to separte multikueue subfolder (#3135)
kubernetes-sigs/kueue: site/content/en/docs/tasks/run/multikueue/jobsets.md

---
title: "Run Jobsets in MultiKueue"
linkTitle: "Jobsets"
weight: 2
date: 2024-09-25
description: >
  Run a MultiKueue scheduled JobSet.
---

## Before you begin

Check the [MultiKueue installation guide](/docs/tasks/manage/setup_multikueue) on how to properly setup MultiKueue clusters.

We recommend using Kueue v0.8.1 or newer and JobSet v0.5.2 or newer.

## Run Jobsets example

The same `jobset-sample.yaml` file from [single cluster environment](docs/tasks/run/jobsets) can be used in a [MultiKueue environment](#multikueue-environment).

> In that setup, the `spec.managedBy` field will be set to `kueue.x-k8s.io/multikueue`

Reviewer: ```suggestion
In that setup, the `spec.managedBy` field is defaulted to `kueue.x-k8s.io/multikueue`
```

> The same `jobset-sample.yaml` file from [single cluster environment](docs/tasks/run/jobsets) can be used in a [MultiKueue environment](#multikueue-environment).

Reviewer: Add intro and "Before you begin" section. It should say that you have MultiKueue setup (referencing the steps).
